app/controllers/application_controller.rb

```diff
@@ -156,4 +150,15 @@ def set_csrf_headers
       cookies['XSRF-TOKEN'] = form_authenticity_token if cookies['XSRF-TOKEN'].blank?
     end
   end
+
+  # protected method to allow access from other controllers
```

Comment:
nice, I could also see moving this to BulkDownloadService

Reply:
f5a85e5